authenticate/handlers.go

```diff
@@ -98,6 +99,7 @@ func (a *Authenticate) mountDashboard(r *mux.Router) {
 	sr.Path("/sign_in").Handler(a.requireValidSignature(a.SignIn))
 	sr.Path("/sign_out").Handler(a.requireValidSignature(a.SignOut))
 	sr.Path("/webauthn").Handler(webauthn.New(a.getWebauthnState))
+	sr.Path("/device-enrolled").Handler(handlers.DeviceEnrolled())
```

Comment:
Should we ensure that the origin of the request is Pomerium (e.g. is valid signed / hmac'd?)

Reply:
It's becoming a little hard to follow all the different handlers getting mounted here. 

We have handlers in this directory, `webauthn` and now /handlers. Thoughts on adding these handler to the existing `webauthn` package?

Reply:
I don't think it belongs there, but sure I can move it.

Reply:
It's okay.